code/datums/dna.dm

```diff
@@ -168,11 +168,13 @@ GLOBAL_LIST_INIT(total_uf_len_by_block, populate_total_uf_len_by_block())
 	//ignores TRAIT_AGENDER so that a "real" gender can be stored in the DNA if later use is needed
 	switch(holder.gender)
 		if(MALE)
-			L[DNA_GENDER_BLOCK] = construct_block(G_MALE, 3)
+			L[DNA_GENDER_BLOCK] = construct_block(G_MALE, 4)
 		if(FEMALE)
-			L[DNA_GENDER_BLOCK] = construct_block(G_FEMALE, 3)
+			L[DNA_GENDER_BLOCK] = construct_block(G_FEMALE, 4)
```

Comment:
Why did these change as well?

Reply:
I'm mostly porting this straight from the source, but if I understand the code to any degree, the number at the end is possible choices, I think.

Reply:
if you don't understand something you shouldn't port it, we've had issues with ports in the past and it's important to know what you're doin.
It's less of an issue here because lexi is around our community, but it's still kinda crummy

Reply:
That's completely fair, I ported it because it seemed low-impact and I notified maintainers when I did so I was hopeful someone could correct me if some piece of the code would magically break everything.

I also did test the changes before putting the PR up, always do.

Reply:
Why is this an argument if it is supposed to be static?